modules/openssh/9.9p1.bcr.1/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  bazel:
+    - 7.x
+    - 8.0.0rc2
+  platform:
```

Comment:
Add MacOS builds? We could filter `sshd` or just build a subset on MacOS until we get `@libxcrypt` on MacOS?

Reply:
Or we wait for #3150, update the dependency and do the full build.

Reply:
Yes. Let’s merge libxcrypt and then upgrade the dep and add the CI check.